src/docs/cookbook-emails/set-up-outgoing-email-settings/index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Set up outgoing email settings
+slug: set-up-outgoing-email-settings
+updated: 2020-12-02
+category: cookbook-emails
+ingress:
+  This article describes how to set up a custom email sender address for
+  your production marketplace.
+published: true
+---
+
+Setting up reliable email delivery is crucial for your marketplace to
+work properly. It is also important to brand your email sending name and
+address to look professional and polished.
+
+Flex uses [SendGrid](https://sendgrid.com/) as an email service
+provider. We manage your SendGrid account and settings automatically,
+and the costs are included in your Flex subscription.
+
+When you have your production environment created and you have bought
+your custom domain for it, you should set up and validate the outgoing
+email settings. Follow the steps below, and you can be sure that your
+emails will be delivered reliably.
+
+## 1. Add sender name and address
+
+Make sure you select your production environment in the top navigation
+in Console. Then head on to the General settings page in the Build
+section. You can see the outgoing email settings below the marketplace
+information:
+
+![Outgoing email settings 1/3](./email-settings-1of3.png)
+
+Click on the "1/3 Start setup" button to start the setup. Follow the
+instructions in the UI.
+
+When you have added your sender email information, you will be prompted
+with DNS information:
+
+![Outgoing email settings 3/3](./email-settings-3of3.png)
+
+At this point, we have created a SendGrid subuser for you, stored the
+information, and authenticated the given sender domain to SendGrid. This
+means that you have allowed this SendGrid subuser to send emails from
+this domain.
+
+## 2. Configure DNS
```

Comment:
Should we link under here to the video instructions that SendGrid provides: https://sendgrid.com/docs/ui/account-and-settings/dns-providers/

Another possibility is to link to here: https://sendgrid.com/docs/ui/account-and-settings/how-to-set-up-domain-authentication/ but I think it's providing a bit too much information.